docs/native-api/QuirkSettings-api-windows.md

```diff
@@ -28,7 +28,7 @@ Runtime setting allowing Networking (HTTP, WebSocket) connections to skip certif
 
 > **EXPERIMENTAL**
 
-By default `react-native-windows` will handle various back events and forward them to JavaScript. Setting this to [`BackNavigationHandlerKind.Native`](BackNavigationHandlerKind) prevents `react-native-windows` from handling these events, including forwarding to JavaScript.  This will allow applications to handle back navigation in native code, but will prevent the `BackHandler` native module from receiving events.
+By default `react-native-windows` will handle various back events and forward them to JavaScript. Setting this to [`BackNavigationHandlerKind.Native`](BackNavigationHandlerKind#native) prevents `react-native-windows` from handling these events, including forwarding to JavaScript.  This will allow applications to handle back navigation in native code, but will prevent the `BackHandler` native module from receiving events.

```

Comment:
```suggestion
By default `react-native-windows` will handle various back events and forward them to JavaScript. Setting this to [`BackNavigationHandlerKind.Native`](BackNavigationHandlerKind) prevents `react-native-windows` from handling these events, including forwarding to JavaScript.  This will allow applications to handle back navigation in native code, but will prevent the `BackHandler` native module from receiving events.

```